tasks/javascript.js

```diff
@@ -113,6 +115,17 @@ gulp.task('javascript:lint', function () {
 
 // karma unit tests in local library only
 gulp.task('javascript:unit', function () {
+  var reporters = cliArgs.karmareporter;
+  if (reporters.constructor === Array) {
+    reporters = Array;
+  }
+  else if (typeof reporters === 'string' &&
+      reporters !== karma.yargsOption.value.default) {
```

Comment:
Necessary to avoid attempting to load the default reporter, until it is extracted to a separate package.